betatest/templates/betatest/welcome_email.txt

```diff
@@ -0,0 +1,26 @@
+{% autoescape off %}
+Hi {{ application.user.profile.full_name }},
+
+we are pleased to inform you that your application to participate in the
+OpenCraft Instance Manager beta test has been accepted.  Your Open edX instance
+has been provisioned and is available at
```

Comment:
"will be available ... soon" or "is available..." but not "is available ... soon"